target/riscv/monitor.c

```diff
@@ -112,7 +147,7 @@ static void walk_pte(Monitor *mon, hwaddr base, target_ulong start,
 
         paddr = (hwaddr)(pte >> PTE_PPN_SHIFT) << PGSHIFT;
 #ifdef TARGET_CHERI
-        attr = pte & (PTE_LC | PTE_SC | 0xff);
+        attr = pte & (PTE_CR | PTE_CW | PTE_CD | PTE_CRM | PTE_CRG | 0xff);
```

Comment:
Did this work on RV32 before? It looks like it shouldn't (or maybe just silently truncates so it's fine?)...